src/views/Incorporation/IncorporationDefineCompany.vue

```diff
@@ -214,7 +214,7 @@ export default class IncorporationDefineCompany extends Vue {
       streetAddressAdditional: ''
     }
 
-    if (this.isTypeBcomp) {
+    if (this.isBaseCompany) {
```

Comment:
And a similar check in OfficeAddresses.vue?

Reply:
Ah, good thought, it's already been removed in a previous commit

Reply:
I don't see this fix in the main branch or in this PR.

Reply:
Is this what you're looking for?  https://github.com/bcgov/business-create-ui/pull/496/files#diff-d2fd1bdae3bfa602eb16d6c4690a9b40068cac64bc566360477b7b562023ae67

Reply:
Doh! Yeah that's it.